src/index.ts

```diff
@@ -9,80 +10,117 @@ require('dotenv').config();
 let success = 0, failure = 0;
 const missingDeps = [];
 let packageScope = [];
+let J1_ACCOUNT, J1_USERNAME, J1_PASSWORD;
 
 (async () => {
   while (true) {
-    var scope = readlineSync.question('Please input a package scope, i.e. @package (input QUIT when finished): ');
+    var scope = readlineSync.question('Input a package scope, i.e. @package (input QUIT when finished): ');
     if (scope === 'QUIT') {
       break;
     }
     packageScope.push(scope);
   }
-  console.log('');
-  const j1Client = await new JupiterOneClient({
-    account: 'j1dev',
-    username: 'j1dev',
-    password: 'dev',
-    poolId: process.env.J1_USER_POOL_ID,
-    clientId: process.env.J1_CLIENT_ID,
-    accessToken: process.env.J1_API_TOKEN,
-    dev: process.env.J1_DEV || false
-  }).init();
+  const clientInput = [];
+  const pathToRepos = readlineSync.question('Input path to directory with repos (relative to root directory): ');
+  clientInput.push(readlineSync.question('Input account. If using a .env file, continue without input: '));
+  clientInput.push(readlineSync.question('Input username. If using a .env file, continue without input: '));
+  clientInput.push(readlineSync.question('Input password. If using a .env file, continue without input: '));
+  clientInput.push(readlineSync.question('Input access token. If using a .env file, continue without input: '));
```

Comment:
Ah, now I see how values could be empty strings. I'm a fan of dealing with that here in the readline codes, and passing `undefined` as values to `getClient`. Also, would be great to go for `const clientInput = { account: sanitizedValue, ... }` instead of a positional array.